src/Compilers/CSharp/Portable/Symbols/ParameterSymbol.cs

```diff
@@ -106,7 +106,9 @@ internal bool IsMarshalAsObject
         public abstract int Ordinal { get; }
 
         /// <summary>
-        /// Returns true if the parameter was declared as a parameter array. 
+        /// Returns true if the parameter was declared as a parameter array.
+        /// Note: it is possible for any parameter in IL to have the [ParamArray] attribute,
```

Comment:
>it is possible for any parameter in IL [](start = 18, length = 38)

I think it is also true for a source parameters improperly marked with params modifier